app/Filament/Resources/DepartmentResource/Pages/EditDepartment.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Filament\Resources\DepartmentResource\Pages;
+
+use App\Filament\Resources\DepartmentResource;
+use Filament\Actions;
+use Filament\Resources\Pages\EditRecord;
+
+class EditDepartment extends EditRecord
+{
+    protected static string $resource = DepartmentResource::class;
+
+    protected function getHeaderActions(): array
+    {
+        return [
+            Actions\DeleteAction::make(),
```

Comment:
@kolanaveen Remove delete action from here as we already given delete action in listing.